react-validated-input-component/my-app/src/ValidatedInput.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import './ValidatedInput.css';
+
+export default function ValidatedInput() {
+  const [password, setPassword] = useState('');
+  const [isValid, setIsValid] = useState(false);
```

Comment:
@j-alejandro-araujo Try not to add values to state that can be entirely calculated based on other props & state, to reduce the chance of introducing bugs with inconsistent state. Instead, calculate that value at render time.

Reply:
Thanks for flagging, it's been adjusted!